usagov/vote-gov-drupal
# VOTE-972: Add robotstxt module. Delete default robots.txt file and pr…

…event if from being committed to repo.



## Jira ticket

[VOTE-972](https://cm-jira.usa.gov/browse/VOTE-972)

## Description

Add contrib module, RobotsTxt, and configure the robots.txt out put for production's static site.

## Deployment and testing

### Post-deploy steps

1. Update your setting.local.php file to set config split environment to production
`$config['config_split.config_split.production']['status'] = TRUE;`
2. Locally run `lando retune`

### QA/Testing instructions

1. Goto /robots.txt and confirm the output contains a list of Allows and Disallows for the environment(enabled config split setting).
2. Run `lando static` to generate static site locally.
3. Confirm that the robots.txt and the correct entries are found in the static directory.
4. Update your setting.local.php file to set config split environment to non_production
`$config['config_split.config_split.non_production']['status'] = TRUE;`
5. Run `lando drush cim` to import non-production configs.
6. Goto /robots.txt and confirm the output is set to block everything from being crawled.
7. Repeat Steps 2-3
8. Repeat Steps 4-7 after updating config split setting to 
`$config['config_split.config_split.local']['status'] = TRUE;`

## Checklist for the Developer




## Checklist for the Peer Reviewers
